src/RMP/Translate/lang/programmes/en.po

```diff
@@ -1702,6 +1727,16 @@ msgstr "Show less"
 msgid "show_more"
 msgstr "Show more"
 
+#. English: "Show More Added Episodes and Clips".
+#. Priority for: Favourites
+msgid "show_more_added_episodes_and_clips"
+msgstr "Show More Added Episodes and Clips"
```

Comment:
Should be sentence case: "Show more added episodes and clips"
If you look around, you'll see all the other translations are too.